hr_attendance_autoclose/data/hr_attendance.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.cron" id="check_attendance_cron">
+        <field name="name">Check Attendance</field>
+        <field name="interval_number">1</field>
+        <field name="interval_type">hours</field>
+        <field name="numbercall">-1</field>
+        <field name="nextcall" eval="(DateTime.now() + timedelta(minutes=60)).strftime('%Y-%m-%d %H:05:00')"/>
```

Comment:
I am having problems with this. The field seems to be computing properly but after some testing it looks like the cron is ignoring this value and executing 1 hour after the last one finished and not at 10:05, 11:05, 12:05 as it should. I don't know if this trick has stopped working on 11.0 or I am doing something wrong.

Reply:
I think this is standard Odoo. The next execution time is calculated when the cron is finished, instead the previous execution time. Hope they change that in future versions.